### ISSUE TITLE
Document the distinctions between active/inactive, orphaned, etc.

### ISSUE THREAD

**Issue by jamesaoverton** [OBOFoundry/OBOFoundry.github.io#1126]
This builds on #733. We've done the work, but don't have clear documentation. Sources:

- discussion in #733 
- the table-sorting code, which distinguished tiers: foundry, library, orphaned, obsolete

**Comment by jamesaoverton** [comment]
This is the most informative bit of https://github.com/OBOFoundry/OBOFoundry.github.io/issues/733#issuecomment-430312155:

> 'active': The ontology project has a contact person who is responsive. Terms are being added or edited in response to community requests.
>
> 'inactive': The ontology project has a contact person who is responsive. A static version of the ontology is available, but no edits are being made.
>
> 'orphaned': The ontology project does not have a contact person who is responsive. A static version of the ontology exists.
>
> 'obsolete': The ontology project is not in active development, and the past developers do not recommend using existing versions of it, either because another project is available that supersedes it, or because previous produced versions have serious issues that make them less usable, and/or are not available at all.

[`util/validate-metadata.py`](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/util/validate-metadata.py#L216) classifies into five levels, which drive the sorting of the main table (although not all distinctions are obvious in the table):

1. foundry: `activity_status: active` and `in_foundry_order: 1` (that's ugly)
2. active: `activity_status: active`
3. inactive: `activity_status: inactive`
4. orphaned: `activity_status: orphaned`
5. obsolete: `activity_status: inactive` and `is_obsolete: true`; the latter should imply the former

**Comment by pbuttigieg** [comment]
That's a pretty good starting point. I think that the foundry/library classification is orthogonal, which is why it could cause confusion. We should separate the two.

Also, we'd need some sort of time threshold (or similar) on an "inactive" status. When will it be considered static? Consider that some small ontologies may be "inactive" because they accomplished their aim and new terms may only come in very occasionally. Being labelled "inactive" gives the wrong impression here.

Suggested revisions:

___

## Ontology Development Status

* active: The ontology project has a contact person who is responsive. Terms, with resolvable IRIs, are being added or other revisions made in response to community requests and/or the aims of the editorial team.

* inactive: The ontology project has a contact person who is responsive. A version of the ontology is available, but no edits are being made and requests for edits are either greatly delayed or not being addressed by the ontology's editors.

* orphaned: The ontology project does not have a contact person who is responsive. A static version of the ontology exists, but no edits are being made and requests for edits are not being addressed or responded to by the ontology's editors.

* obsolete: The ontology is either inactive or orphaned, and the original developers do not recommend using existing versions of it, either because another project is available that supersedes it, or because previous produced versions have serious issues that make them less usable, and/or are not available at all.

## OBO Alignment Status

* Foundry: The ontology has undergone and passed thorough review by the OBO Operations Committee, confirming its alignment to the OBO Principles. 

* Library: The ontology has been issued a namespace and integrated into the OBO registry following an initial vetting of its alignment to the OBO Principles. 

___

**Comment by mellybelly** [comment]
I think that having the ontologies organized by Foundry status is disadvantageous to OBO. I continue to hear that people come to the OBO page and disregard the ontologies that are not "Foundry"; e.g. the OBO Foundry does not have many production ontologies." However, many of the non-Foundry ontologies have had extensive review and adoption in the community outside the context of the OBO committee review. 

All of the ontologies in the library aim to adhere to the principles and this is evaluated in the context of their request. Meanwhile, a dated Foundry review admitting an ontology into the Foundry does not indicate that the ontology is of better quality. IMHO, being a Foundry ontology only means that there has been an internal OBO review, it is not an indication of quality, interoperability, or any other functionality.

I suggest that the OBO reviews become a linked attribute in the table, similar to other attributes. The reviews should also be dated.

**Comment by jamesaoverton** [comment]
@pbuttigieg We haven't set time threshold for "inactive". When @rvita was emailing all contacts over the past year, some voluntarily classified their projects as inactive, so we had to add this category (see the rest of #733). There's been discussion of scheduled manual reviews of project status, but no firm plans.

Foundry status is not orthogonal, because foundry projects must be active.

@mellybelly This issue is about documenting existing metadata on project status. We're doing a ton of work on automated tests for "quality, interoperability, or any other functionality" with the OBO Dashboard, and there are other open issues about dated reviews.

**Comment by cmungall** [comment]
Actually we don't have a dedicated ticket about dated reviews AFAICT. I added one: #1140

Regarding:
> I suggest that the OBO reviews become a linked attribute in the table, similar to other attributes. The reviews should also be dated.

We have this ticket that @wdduncan is working on: #1088

**Comment by yongqunh** [comment]
I agree with @pbuttigieg. Some ontology may be inactive due to their maturation or lack of more feature request or use cases for further improvement. Different situations may need to be considered.

**Comment by jamesaoverton** [comment]
See also #300 about hiding obsolete ontologies by default.

**Comment by cmungall** [comment]
I note that orphaned status isn't showing up on the site

http://obofoundry.org/ontology/rex

should show up as orphaned

**Comment by yongqunh** [comment]
REX: an ontology of physico-chemical processe.  It's different from Orphaned, right?

**Comment by cmungall** [comment]
I think you mean Orphanet? I am talking about the status field!

**Comment by yongqunh** [comment]
I see. thanks for explaining.

**Comment by nlharris** [comment]
are there still things to be done here?

**Comment by matentzn** [comment]
For now this is enough:

https://obofoundry.org/docs/OntologyStatus.html

We can iterate in smaller portions. We are waiting for @ddooley and his team to provide a new front page for sorting ontologies better.
